frontend/dockerfile/linter/ruleset.go

```diff
@@ -148,4 +148,12 @@ var (
 			return fmt.Sprintf("Default value for ARG %v results in empty or invalid base image name", baseName)
 		},
 	}
+	RuleCopyIgnoredFile = LinterRule[func(string) string]{
+		Name:        "CopyIgnoredFile",
+		Description: "File is ignored by .dockerignore",
+		URL:         "",
+		Format: func(file string) string {
+			return fmt.Sprintf("File %q is ignored by .dockerignore", file)
```

Comment:
Ideally we could show the pattern that causes the issue but that might be tricky with current libs so can be left for follow-ups.

Reply:
Updated with more detail, and added the needs/follow-up flag. :)